.github/workflows/test.yml

```diff
@@ -35,6 +35,7 @@ jobs:
       matrix:
         type:
           [
+            shellcheck,
```

Comment:
I have the impression that for many linting jobs the setup on GitHub CI will take more time than the execution of the linter.

Mind checking if this is true for shellcheck? If so, I suggest modifying make lint to also execute this one.

Reply:
That's a good point. I'll put `shellcheck` into the `lint` target. 👍 

Regardless, all the matrix jobs should run in parallel, assuming GitHub resources are plentiful as usual, and I doubt this would be slower than any of the others.

Reply:
Done. There's a dedicated `lint-shell` target that is now called from `lint` like the others, and the existing `lint` GitHub job includes this now.

In the `lint` target definition, I put `lint-shell` before `lint-ruff` because `shellcheck` is faster than `ruff` because it's only checking two small files. 🦔💨